Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -2180,6 +2180,11 @@ public virtual async Task OnPipCompleted(RunnablePip runnablePip)
                         PipExecutionCounters.AddToCounter(PipExecutorCounter.ProcessDuration, processDuration);
                         m_groupedPipCounters.IncrementCounter(processRunnablePip.Process, PipCountersByGroup.Count);
                         m_groupedPipCounters.AddToCounter(processRunnablePip.Process, PipCountersByGroup.ProcessDuration, processDuration);
+
+                        if(!succeeded && result.Status == PipResultStatus.Failed)

```

Comment:
>!succeeded [](start = 27, length = 10)

You can remove the succeeded part of the check, It turns out that result.Status.InidicatesFailure() is just:
            return result == PipResultStatus.Failed || result == PipResultStatus.Skipped || result == PipResultStatus.Canceled;

so it's a looser check than the second == PipResultStatus.Failed